### DISCUSSION TITLE
Add the ability  to scale images within stepper blocks and expandable blocks

### DISCUSSION THREAD

**Discussion by Olga-tech-writer** [GitbookIO/community#826] [Feature Requests]
Currently, it's not possible to adjust the size of images within stepper blocks, resulting in their display being too small despite the high definition of the original images. The same limitation applies to images within expandable blocks. Can you add the ability to scale images within those blocks?

**Comment by aikoii** [comment]
Hey @Olga-tech-writer! You should now be able to scale images within stepper blocks 🎉.

![CleanShot 2024-12-12 at 22 47 10@2x](https://github.com/user-attachments/assets/1a936ff4-d41b-4b0a-aca0-188faca8ba0c)

**Reply by Olga-tech-writer** [reply]
Thanks, that works. Due to our cofiguration, we can make changes only via Markdown (not UI), and I incorrectly passed the width parameter for images.
